src/main/kotlin/no/nav/syfo/kafka/varselbus/domain/EsyfovarselHendelse.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.kafka.varselbus.domain
+
+import java.io.Serializable
+
+data class EsyfovarselHendelse(
+    val mottakerFnr: String,
+    val type: HendelseType,
+    var data: Any?
+) : Serializable
+
+enum class HendelseType {
+    NL_OPPFOLGINGSPLAN_SENDT_TIL_GODKJENNING,
+    NL_OPPFOLGINGSPLAN_OPPRETTET
+}
+
+fun HendelseType.toDineSykmeldteHendelse(): DineSykmeldteHendelse {
+    return when (this) {
+        HendelseType.NL_OPPFOLGINGSPLAN_SENDT_TIL_GODKJENNING -> DineSykmeldteHendelse.OPPFOLGINGSPLAN_TIL_GODKJENNING
+        HendelseType.NL_OPPFOLGINGSPLAN_OPPRETTET -> DineSykmeldteHendelse.OPPFOLGINGSPLAN_OPPRETTET
+    }
+}
```

Comment:
Jeg synes disse hører hjemme i DineSykmeldteHendelse.kt

Reply:
Eller kanskje i en egen fil. Ikke helt sikker.

Reply:
Jeg så feil på navnene, siden det er to klasser med samme navn. Renamer den ene